ext/mysqli/tests/bug81335.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+Bug #81335: Packets out of order after connection timeout
+--EXTENSIONS--
+mysqli
+--SKIPIF--
+<?php
+if (getenv("SKIP_SLOW_TESTS")) die("skip slow test");
+
+if (!defined('MYSQLI_STORE_RESULT_COPY_DATA')) die('skip requires mysqlnd');
```

Comment:
You are making this change against master branch, so mysqlnd is the only option available. This check is redundant and will possibly break in PHP 9.0 once the constant is removed. 

The CI jobs are failing. Could you please check why?

Reply:
Should I make this MR for 8.3?

Reply:
I will check the reason for the CI failure on windows later.

Reply:
I think this could even go onto 8.2.

Reply:
OK, I will update later.

Reply:
> The CI jobs are failing. Could you please check why?
This does not reproduce on my local test, I am not sure why.

Reply:
> I think this could even go onto 8.2.

The MR target 8.2 now.